todolist/app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "todolist",
+  "displayName": "todolist"
+}
```

Comment:
I don't know if you have it but you should set in Visual Code to automatically add a empty line after every file